lib/timescaledb/database/schema_statements.rb

```diff
@@ -163,6 +222,18 @@ def cast_create_hypertable_optional_arguments(options)
           end
         end.compact
       end
+
+      # @param [Array<Hash<Symbol, Object>>] options The continuous aggregate policy arguments.
+      # @return [Array<String>]
+      def cast_continuous_aggregate_policy_arguments(options)
+        options.map do |option, value|
+          case option
+          when :if_not_exists then "#{option} => #{boolean_to_sql(value)}"
+          when :start_offset, :end_offset, :schedule_interval then "#{option} => #{value.nil? ? 'NULL' : interval_to_sql(value)}"
+          when :initial_start, :timezone then "#{option} => #{quote(value)}"
+          end
+        end.compact
```

Comment:
I don't get why we need the compact here. Any special reason?

Reply:
It's just in case they pass an invalid option, in that case we will end with a nil value here and that would break the queries that call these helper methods

Reply:
Got it! very good!